crystal/distribution.yaml

```diff
@@ -401,6 +401,21 @@ repositories:
       url: https://github.com/stonier/ecl_tools.git
       version: release/1.0-crystal
     status: maintained
+  eigen_stl_containers:
+    doc:
+      type: git
+      url: https://github.com/ros/eigen_stl_containers.git
+      version: ros2
+    release:
+      tags:
+        release: release/crystal/{package}/{version}
+      url: git@github.com:ros2-gbp/eigen_stl_containers-release.git
```

Comment:
@nuclearsandwich I really wanted this to be `https://github.com/ros2-gbp/eigen_stl_containers-release.git`, but due to 2FA, I had to enter this URL to push/pull stuff.  Should I leave this as-is, or modify by hand?

Reply:
Let's modify it by hand. In the future you can use the bloom argument `--override-release-repository-push-url git@ssh.url`

If you do this a lot then creating aliases can be useful. Here are mine in fish.

```fish
function crystalnew
	set reponame $argv[1]
	bloom-release -n -r crystal $reponame --override-release-repository-push-url git@github.com:ros2-gbp/$reponame-release
end


function crystalrev
	set reponame $argv[1]
	bloom-release -r crystal $reponame --override-release-repository-push-url git@github.com:ros2-gbp/$reponame-release
end
```

Technically those are metaprogrammed with a makefile and a for loop to cover every distro but this is far more illustrative.

Reply:
Updated release repo git url with 33d37fb.

Reply:
Thanks for that, didn't know about the overrides.  Will use that in the future.